src/index.css

```diff
@@ -1,5 +1,9 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap');
 
+* {
+ font-family: 'Poppins', 'Arial', 'sans-serif' !important;
```

Comment:
Great idea to add the font also to the storybook implementation.
But I would rather be carefull with this. With this code being in the index.css file, it would also be compiled in the build for the npm package.

I would recommend to remove the code from here. To have the same affect, we can create a "storybook.css" file with this code in the ".storybook" folder and import this code in the preview.js file. This means that we have a page wide styling with these fonts for our storybook, but will not be shipped in the npm package.

Reply:
@malinovic could not make this work without breaking other things. I have removed the code in the index.css and will create an issue to add the font to storybook.

Reply:
Okay, thanks. I will consider this review comment as resolved then.